### DISCUSSION TITLE
METviewer bug when loading xml for rank histogram

### DISCUSSION THREAD

**Discussion by gsketefian** [dtcenter/METplus#2486] [Plot Generation] [answered]
Hi,

I'm using the METviewer GUI at https://dtcenter.ucar.edu/met/metviewer/servlet?jsp=new to make rank histograms (rhist).  (The GUI is currently based on METviewer 5.1.0, so it's possible this issue has been resolved in a newer version.  If so, it would be great to know how to access that version using a web browser.)

I find that while I can create an rhist in the GUI from scratch, I'm not able to load an existing xml that plots an rhist even though that xml can be run without issues using the METviewer batch script on mohawk.

I know others at DTC have run into the same issue.  I could not find an existing discussion for this, so I'm starting this one.

The problem appears to be a mangled pull-down menu for `MODEL` under `Series Variables` in the top left section of the GUI.  Loading the xml (attached) causes the model names in the database in the pull-down menu next to `MODEL` to be unclickable.

The same thing happens if I successfully create an rhist from scratch in the GUI, then save the resulting xml, and then try to load it back into the GUI.  The reloading process gives me the mangled `MODEL` pull down menu.

I've attached a sample xml that you can use for testing (you have to remove the `.txt` extension first; I had to add it to be able to upload).

Thanks for any help you can provide.
Gerard

[plot_rhist_DPT2m_nostoch_stoch.xml.txt](https://github.com/dtcenter/METplus/files/14184763/plot_rhist_DPT2m_nostoch_stoch.xml.txt)

**Comment by TatianaBurek** [comment, marked as answer]
I was able to reproduce the bug during XML loading.
I created an issue : https://github.com/dtcenter/METviewer/issues/513

Currently we don't have a development version of the METplus Analysis tools installed and exposed on NCAR server. @JohnHalleyGotway do you think it would be useful to create and maintain another instance of tools on mohawk that would be updated after each beta release? We can discuss this during the next METplus Analysis Tools meeting

**Reply by jprestop** [reply]
I'm following up on Discussions.  It looks like this work has been done and the issue listed above has been closed.  If there is any further discussion needed, please post a comment on the issue.
